.gitignore

```diff
@@ -67,6 +67,7 @@ Thumbs.db
 /.vs*
 /*.local.bat
 /CMakeSettings.json
+/.cache
```

Comment:
It is from VSCode plugin I've been using, I think it is okay to have it here, but not in separate PR(but let me know if it is not)